test/graphtile.cc

```diff
@@ -52,6 +52,9 @@ void id_from_string() {
   if(GraphTile::GetTileId("foo2/8675309/bar/1baz2/qux42corge/3/001/000/002.gph") != GraphId(1000002,3,0))
     throw std::logic_error("Unexpected graphtile id");
 
+  if(GraphTile::GetTileId("foo2/8675309/bar/1baz2/qux42corge/3/001/000/002") != GraphId(1000002,3,0))
```

Comment:
unrelated but it seemed like we should at least test that not hvaing an extension is ok (which it should be)